apm-agent-plugins/apm-ratpack-plugin/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-agent-plugins</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.7.0</version>
```

Comment:
```suggestion
        <version>1.8.1-SNAPSHOT</version>
```

Reply:
I merged in v1.8.0 for the moment. I'm testing in production, so wanted to make sure I'm on a stable release. I'll rebase to master and apply the suggestion once I sort a couple things out. Hopefully this week.